packages/gatsby-theme-docs/src/layouts/internals/layout-page-navigation.js

```diff
@@ -134,7 +134,7 @@ const LayoutPageNavigation = props => {
         <SlidingContainer>{navigationContainer}</SlidingContainer>
       </ContainerOverlay>
 
-      <GridContainer>
+      <GridContainer role="page-navigation">
```

Comment:
I'm not aware of an ARIA role "page-navigation".   There's only "navigation" if you are not using the `<nav>` tag.  I think what you want here is an aria-label "Page Navigation".

Reply:
Right, let me change it to `navigation` and provide a `aria-label` for differentiate between the 2 navs

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role#Labeling_landmarks